src/libraries/System.Threading.ThreadPool/ref/System.Threading.ThreadPool.csproj

```diff
@@ -3,6 +3,10 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
     <Nullable>enable</Nullable>
+    <FeatureWasmThreads>false</FeatureWasmThreads>
```

Comment:
nit: do you need this to be set to false? The above condition only checks on true and that's usually sufficient in the msbuild world.

Reply:
I think I like it as documentation and a reminder to flip it over to `true` once this stuff is no longer experimental, or once we have a [better comptability analyzer](https://github.com/dotnet/designs/pull/261)